Sources/ImageProcessor.swift

```diff
@@ -167,25 +167,48 @@ public struct RoundCornerImageProcessor: ImageProcessor {
 
 /// Processor for resizing images. Only CG-based images are supported in macOS.
 public struct ResizingImageProcessor: ImageProcessor {
+    public enum ContentMode {
+        case none
+        case aspectFit
+        case aspectFill
+    }
+    
     public let identifier: String
     
     /// Target size of output image should be.
     public let targetSize: CGSize
     
+    /// Target content mode of output image should be.
+    /// Default to ContentMode.none
+    public let targetContentMode: ContentMode
+    
     /// Initialize a `ResizingImageProcessor`
     ///
     /// - parameter targetSize: Target size of output image should be.
+    /// - parameter contentMode: Target content mode of output image should be.
     ///
     /// - returns: An initialized `ResizingImageProcessor`.
-    public init(targetSize: CGSize) {
+    public init(targetSize: CGSize, contentMode: ContentMode = .none) {
         self.targetSize = targetSize
-        self.identifier = "com.onevcat.Kingfisher.ResizingImageProcessor(\(targetSize))"
+        self.targetContentMode = contentMode
+        self.identifier = "com.onevcat.Kingfisher.ResizingImageProcessor(\(targetSize), \(targetContentMode))"
```

Comment:
`identifier` is used to identify the post-processed images in cache, so we need to keep the `identifier` the `same` for `.none` case.